docs/interactions/structured-messages/select.md

```diff
@@ -165,6 +165,7 @@ Primary parameters are used by default, however, some parameters are unique or o
 | **`items.title`** | String | The title of the item. *Truncated to 20 characters.* |
 | **`items.attachment_id`** | String | **Optional**. Existing attachment id used to decorate the item in the list. Supports private attachments. [Upload attachments](../../../basics/uploads) for you own custom images. |
 | **`items.attachment_fallback_id`** | String | **Optional**. Fallback in case the attachment related to the attachment_id doesn’t meet the source requirements. Must be public. Only jpg, jpeg, png formats. Maximum size of 5 MB. [Upload attachments](../../../basics/uploads) for you own custom images. |
+| **`items.payload`** | String | **Optional**. The item payload field. Limited to 200 characters. Automatically gets populated as a random hex if blank. |
```

Comment:
1. Why did you add back it's optional and it's the item payload field? It's already said in the generic table, here it's about specificities
2. It's not limited to 200 chars, only WA is.
3. Can you also handle the other strmes type? Like template and carousel.

Reply:
1. I just copied the text from WA and wanted it to be consistant with the other sources
2. OK didn't know about that thank you
3. yes OK